### PR TITLE
mail-filter/rspamd: update to version 1.9.4

### PR DESCRIPTION
**Pull request by rseichter**
@croessner I hope you don't mind me simply copying your 1.9.0 ebuild. Upstream releases for Rspamd are so frequent, it is hard to keep up, and the author does not support "older" (as he calls them) releases.

### CODE REVIEW

File: mail-filter/rspamd/rspamd-1.9.4.ebuild [hunk 1]
```diff
@@ -0,0 +1,96 @@
+# Copyright 1999-2019 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=6
```
Comment (mgorny) [inline mail-filter/rspamd/rspamd-1.9.4.ebuild]: Is there anything blocking EAPI 7?

File: mail-filter/rspamd/rspamd-1.9.4.ebuild [hunk 2]
```diff
@@ -0,0 +1,96 @@
...
+
+pkg_setup() {
+	enewgroup rspamd
+	enewuser rspamd -1 -1 /var/lib/rspamd rspamd
+}
+
+src_prepare() {
+	cmake-utils_src_prepare
+
+	sed -i -e 's/User=_rspamd/User=rspamd/g' \
```
Comment (mgorny) [inline mail-filter/rspamd/rspamd-1.9.4.ebuild]: Wouldn't it fit on a single line?

File: mail-filter/rspamd/rspamd-1.9.4.ebuild [hunk 3]
```diff
@@ -0,0 +1,96 @@
...
+		rspamd.service \
+		|| die
+}
+
+src_configure() {
+	local mycmakeargs=(
+		-DCONFDIR=/etc/rspamd
```
Comment (mgorny) [inline mail-filter/rspamd/rspamd-1.9.4.ebuild]: Some `${EPREFIX}` love maybe, while at it?

File: mail-filter/rspamd/rspamd-1.9.4.ebuild [hunk 4]
```diff
@@ -0,0 +1,96 @@
...
+		-DENABLE_GD=$(usex gd ON OFF)
+		-DENABLE_PCRE2=$(usex pcre2 ON OFF)
+		-DENABLE_JEMALLOC=$(usex jemalloc ON OFF)
+		-DENABLE_HYPERSCAN=$(usex cpu_flags_x86_ssse3 ON OFF)
+		-DENABLE_TORCH=$(usex torch ON OFF)
+	)
+	cmake-utils_src_configure
+}
+
+src_test() {
```
Comment (mgorny) [inline mail-filter/rspamd/rspamd-1.9.4.ebuild]: Isn't that redundant?

**Comment by croessner** [comment]
I already have the ebuild ready for the upcoming version 2.0, which drops torch and GD. Also the openrc file is still not correct. I have fixed that as well. Please be a little patience.

**Comment by rseichter** [comment]
Given the release frequency of Rspamd and the unwillingness of the author to support "older" releases (he usually simply closes issues without comment), I don't see how patience would help. I opened this PR a month ago, and the latest available Gentoo ebuild version is 1.9.0. There have been four upstream releases since. 1.9.4 is basically a requirement, and whenever 2.0 is released, it will have to go through the whole PR process again.

@mgorny Is it possible to process this pull request as it stands now (technically it works), and leave the changes for whenever 2.0 comes around?

File: mail-filter/rspamd/rspamd-1.9.4.ebuild (hunk 1 above)
Comment (rseichter) [reply mail-filter/rspamd/rspamd-1.9.4.ebuild]: I don't think so, but I'll leave that to Christian. I have made the other changes you requested, though.

**Comment by mgorny** [comment]
Let's not let perfect be the enemy of good. I'm going to merge this interim version, as I see no reason to block that while 2.0 is being worked on.

**Pull request closed without merging**